docker/iml-agent-comms.dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM rust:1.39 as builder
+WORKDIR /build
+COPY . .
+RUN cd iml-agent-comms && cargo build --release
+
+FROM rust:1.39
+COPY --from=builder /build/target/release/iml-agent-comms /usr/local/bin
+RUN apt-get update \
```

Comment:
I don't think we use postgres in this service.

Reply:
I removed it and rebuilt and it looks like it's using it:
```
 iml-agent-comms_1    | /usr/local/bin/wait-for-dependencies.sh: line 35: psql: command not found
```